config/solidity.ts

```diff
@@ -28,7 +28,7 @@ const PRODUCTION_SOLC_CONFIG: SolcUserConfig = {
     viaIR: true,
     optimizer: {
       enabled: true,
-      runs: 4980,
+      runs: 3000,
```

Comment:
had to reduce optimizer runs to make the Market contract bytecode, which now has additional endpoints for special orders, small enough to be deployable.

this change in optimizer settings is also why the other contracts were upgraded -- because their bytecode changed too